upstream-sha256: a382df9d9755df470439c97bc6d005fe960651a6c20512e422bdd83df471f407
actual-sha256: 476e084f598f4ada930273b73aabf46c3f451143c89046fd4c154ed44ff25343
--- upstream.expected
+++ tolk.actual
@@ -28,12 +28,12 @@
   half4 val9 = (*((half4*)((data1_256+alu3))));
   int alu4 = ((alu0<<7)+alu1);
   float8 wmma0 = __WMMA_16_16_16_half_float(make_half8(val9.x,val9.y,val9.z,val9.w,val8.x,val8.y,val8.z,val8.w), make_half8(val7,val2,val5,val6,val0,val1,val3,val4), make_float8(0.0f,0.0f,0.0f,0.0f,0.0f,0.0f,0.0f,0.0f));
-  *(data0_256+(alu4+112)) = wmma0[7];
-  *(data0_256+(alu4+16)) = wmma0[1];
-  *(data0_256+(alu4+32)) = wmma0[2];
-  *(data0_256+(alu4+48)) = wmma0[3];
-  *(data0_256+(alu4+64)) = wmma0[4];
-  *(data0_256+(alu4+80)) = wmma0[5];
-  *(data0_256+(alu4+96)) = wmma0[6];
-  *(data0_256+alu4) = wmma0[0];
+  *(data0_256+(alu4+112)) = (0.0f+wmma0[7]);
+  *(data0_256+(alu4+16)) = (0.0f+wmma0[1]);
+  *(data0_256+(alu4+32)) = (0.0f+wmma0[2]);
+  *(data0_256+(alu4+48)) = (0.0f+wmma0[3]);
+  *(data0_256+(alu4+64)) = (0.0f+wmma0[4]);
+  *(data0_256+(alu4+80)) = (0.0f+wmma0[5]);
+  *(data0_256+(alu4+96)) = (0.0f+wmma0[6]);
+  *(data0_256+alu4) = (0.0f+wmma0[0]);
 }
